libs/net/entities/Content.cs

```diff
@@ -252,9 +252,9 @@ private Content(string uid, string headline, ContentType contentType, License li
         this.Uid = uid ?? throw new ArgumentNullException(nameof(uid));
         this.Headline = headline;
         this.ContentType = contentType;
-        this.LicenseId = license?.Id ?? throw new ArgumentNullException(nameof(license));
+        if (license != null) this.LicenseId = license.Id;
         this.License = license;
-        this.ProductId = product?.Id ?? throw new ArgumentNullException(nameof(product));
+        if (product != null) this.ProductId = product.Id;
```

Comment:
You can't do this `ProductId` is a required property.  It cannot default to `0`.

Reply:
The same comment as mentioned below, thanks